app/Http/Requests/StoreClass.php

```diff
@@ -0,0 +1,80 @@
+<?php
```

Comment:
What exactly is this class? What is it validating because `Store` is extremely vague. Can I use this to validate instructor data to store? How about class data? I'm sure it can handle students as well?

Reply:
I'm realizing maybe the naming convention I used is wrong. But it's basically this: https://laravel.com/docs/5.5/validation#form-request-validation. It just extends the base request class to do validation outside of the controller.